docs/UserGuide.md

```diff
@@ -193,8 +193,8 @@ you actually think that the module was the best you'd taken in NUS.
 The `edit` command allows you to edit the details of a module that you have added into GradPad.
 
 You can edit a module by specifying the module code of the module in the Completed Modules list, followed by the
-fields you wish to edit with their respective prefixes, 'c/' for module code, 't/' for tags. You may edit multiple fields in a
-single `edit` command.
+fields you wish to edit with their respective prefixes, `c/` for new module code, `t/` for new tag. You may edit multiple fields in a
+single `edit` command. 
```

Comment:
Do yall think that this is clear enough for the reader? It already made sense to me when I only add the word `new`.